src/main/java/de/komoot/photon/nominatim/NominatimConnector.java

```diff
@@ -469,6 +471,16 @@ private void completePlace(PhotonDoc doc) {
                 continue;
             }
 
+            if (address.isNeighbourhood() && doc.getNeighbourhood() == null) {
+                doc.setNeighbourhood(address.getName());
+                continue;
+            }
+
+            if (address.isSuburb() && doc.getSuburb() == null) {
+                doc.setSuburb(address.getName());
+                continue;
+            }
```

Comment:
I wouldn't recommend using the data from Nominatim's computed address at the moment because it is not reliable. Really only set suburb if it shows up in the address tags.